bin/job-template.sbatch

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+#SBATCH --job-name=__NAME__
+#SBATCH --output=output.txt
+#SBATCH --ntasks=__NODES__
+#SBATCH --nodes=__NODES__
+#SBATCH --gres=gpu:__GPUS__
+
+PWD=`pwd`
+compute_cmd="/bin/bash .compute"
+srun="srun --exclusive -N1 -n1"
+
+# unfolding slurm's compact cluster representation
+nodes_raw=`scontrol show hostname $SLURM_JOB_NODELIST`
+index=0
+for node in $nodes_raw
+do
+  # keeping nodes as array for later index lookup
+  nodes[$index]=$node
+  ((index=index + 1))
+done
+# comma separated node list (with leading comma)
+raw_node_list=$(printf ",%s" "${nodes[@]}")
+
+# exporting COMPUTE variables for the .compute script that will
+# get executed on every node of the allocated cluster
+export COMPUTE_NODES=${raw_node_list:1}
+export COMPUTE_NODES_COUNT=${#nodes[@]}
+export COMPUTE_ID="__ID__"
+export COMPUTE_NAME="__NAME__"
+export COMPUTE_JOB_NUMBER=$SLURM_JOB_ID
+export COMPUTE_DATA=/data/shared
+export COMPUTE_RESULTS=$PWD/results
+export COMPUTE_KEEP=$PWD/results/keep
```

Comment:
Would prefer if these three variables had a _DIR suffix (or something along those lines) so it's immediately obvious that they're meant to be directories and nothing else.